src/Compatibility/ControlGallery/src/Android/Compatibility.ControlGallery.Android.csproj

```diff
@@ -39,10 +39,10 @@
     <PackageReference Include="NUnit" Version="3.13.3" />
     <PackageReference Include="Xam.Plugin.DeviceInfo" Version="4.1.1" />
     <PackageReference Include="Xamarin.Build.Download" Version="0.11.4" />
-    <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.6.1.3" />
-    <PackageReference Include="Xamarin.AndroidX.Browser" Version="1.5.0.3" />
-    <PackageReference Include="Xamarin.AndroidX.Palette" Version="1.0.0.19" />
-    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.9.0.2" />
+    <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.8.3" />
+    <PackageReference Include="Xamarin.AndroidX.Browser" Version="1.8.0.3" />
+    <PackageReference Include="Xamarin.AndroidX.Palette" Version="1.0.0.26" />
+    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.11.0" />
```

Comment:
Can all of these version numbers go in `AndroidX.targets`? That way we can be sure we have the same number for each one in a single location.

The way it should work, is omit `Version="1234"` and then `<PackageReference Update="..." Version="1234" />` would set it later in the build.